arm/Microsoft.Storage/storageAccounts/blobServices/deploy.bicep

```diff
@@ -27,24 +27,25 @@ module pid_cuaId '.bicep/nested_cuaId.bicep' = if (!empty(cuaId)) {
 
 resource storageAccount 'Microsoft.Storage/storageAccounts@2021-06-01' existing = {
   name: storageAccountName
+}
 
-  resource blobServices 'blobServices@2021-06-01' = {
-    name: name
-    properties: {
-      deleteRetentionPolicy: {
-        enabled: deleteRetentionPolicy
-        days: deleteRetentionPolicyDays
-      }
-      automaticSnapshotPolicyEnabled: automaticSnapshotPolicyEnabled
+resource blobServices 'Microsoft.Storage/storageAccounts/blobServices@2021-06-01' = {
+  name: name
+  parent: storageAccount
+  properties: {
+    deleteRetentionPolicy: {
+      enabled: deleteRetentionPolicy
+      days: deleteRetentionPolicyDays
     }
+    automaticSnapshotPolicyEnabled: automaticSnapshotPolicyEnabled
   }
 }
 
 module blobServices_container 'containers/deploy.bicep' = [for (container, index) in containers: {
   name: '${deployment().name}-Storage-Container-${index}'
```

Comment:
'${uniqueString(deployment().name, location)}-storage-container-${index}' ?

Reply:
Why? :D Did we not agree to just use the unique string on the top-level?